config/locales/en.yml

```diff
@@ -111,8 +111,8 @@ en:
       You will receive an email with instructions on how to reset your password
       in a few minutes.
     send_code:
-      voice: You will be called with your one-time passcode.
-      sms: Your one-time passcode has been sent via text message.
+      voice: Your one-time passcode was re-sent at %{time} via phone call.
+      sms: Your one-time passcode was re-sent at %{time} via text message.
```

Comment:
This text is misleading. It implies that the same code was resent when clicking "Send a new one" when authenticating. This does not match the fact that a new code gets send each time the link is clicked. The only scenario in which the same code gets sent is during phone confirmation (when setting up a phone for the first time, or when updating an existing phone number).

To make things consistent and simpler, I propose always sending a new code, which is what I was planning on doing as part of my refactor.

Also, `re-sent` is misspelled. It should be `resent`.